ts/features/lollipop/saga/index.ts

```diff
@@ -9,8 +9,17 @@ import {
   deletePreviousCryptoKeyPair
 } from "../../../sagas/startup/generateCryptoKeyPair";
 import { isLollipopEnabledSelector } from "../../../store/reducers/backendStatus";
+import { backendStatusLoadSuccess } from "../../../store/actions/backendStatus";
+import { backendStatusSelector } from "./../../../store/reducers/backendStatus";
 
 export function* generateLollipopKeySaga() {
+  // We must wait for the backend status to be loaded
+  // to be sure to manage alreadey logged in users
+  // coming from old app versions if lollipop is enabled.
+  const backendStatus = yield* select(backendStatusSelector);
```

Comment:
We are not using the backend status value.
It's a little strange that from the "backendStatusLoadSuccess" we infer that we are "sure to manage already logged in users coming from old app versions if lollipop is enabled".

Reply:
Maybe the content of comment is wrong and what we should convey is that "We must wait for the backend status to be retrieved in order to know if the lollipop remote feature flag is enabled (to properly handle the key creation for already-logged-in-but-without-a-generated-public/private-key-pair users later)"?

Reply:
Yep, the problem here was that in this saga we check if the remote lollipop flag is enable to properly manage the already logged in users from a previous app version. This user could not have a `publicKey` or even not have the `keyTag`.

Reply:
What about "We must wait to retrieve the backend status to know if the lollipop remote flag is enabled to properly manage the already logged in users from a previous app version. This users could not have a publicKey or even not have the keyTag."

 The startup saga and the backend status saga are called in parallel.

Reply:
"We must wait to retrieve the backend status to know if the lollipop remote flag is enabled **and** properly manage the already logged in users from a previous app version. **These** users **may** not have a publicKey **nor** a keyTag**, so if lollipop is enabled, we have to generate both keyTag and publickKey**"

Is it clearer?

Reply:
If we eliminate the guard on `isLollipopEnabledSelector` a few lines down, we could dispense with the synchronization aspect involving the backend. We would generate a key irrespective of the remote flag. `isLollipopEnabledSelector` was originally designed for the first phase of Lollipop for the key generation part.

Reply:
Good for me